doc/getting-started/getting-started.rst

```diff
@@ -1,12 +1,23 @@
 .. _rt_industry_ubuntu_setup:
 
-Getting Started Guide for ACRN Industry Scenario With Ubuntu Service VM
-#######################################################################
+Getting Started Guide
```

Comment:
While changing the title is a good idea, changing the filename introduces a problem for external links to the ACRN documentation.  It takes a while for the google search indexer to catch up with this change, and until then, searches will link to the old file name.  Similarly, links from external websites to a specific page will break if we change the filename.  We do have a workaround though, you can add a line in the conf.py to build a redirect from the old filename to the new one.  I'll do that.